### ISSUE TITLE
Auto-DeepLab: Hierarchical Neural Architecture Search for Semantic Image Segmentation

### ISSUE THREAD

**Issue by sile** [sile/readings#81]
URL
-----

https://arxiv.org/abs/1901.02985

Abstract
----------

> Recently, Neural Architecture Search (NAS) has successfully identified neural network architectures that exceed human designed ones on large-scale image classification. In this paper, we study NAS for semantic image segmentation. Existing works often focus on searching the repeatable cell structure, while hand-designing the outer network structure that controls the spatial resolution changes. This choice simplifies the search space, but becomes increasingly problematic for dense image prediction which exhibits a lot more network level architectural variations. Therefore, we propose to search the network level structure in addition to the cell level structure, which forms a hierarchical architecture search space. We present a network level search space that includes many popular designs, and develop a formulation that allows efficient gradient-based architecture search (3 P100 GPU days on Cityscapes images). We demonstrate the effectiveness of the proposed method on the challenging Cityscapes, PASCAL VOC 2012, and ADE20K datasets. Auto-DeepLab, our architecture searched specifically for semantic image segmentation, attains state-of-the-art performance without any ImageNet pretraining.

Table of Contents
--------------------

- (1) Introduction
- (2) Related Work
  - Semantic Image Segmentation
  - Neural Architecture Search Method
  - Neural Architecture Search Space
- (3) Architecture Search Space
  - (3-1) Cell Level Search Space
  - (3-2) Network Level Search Space
- (4) Methods
  - (4-1) Continuous Relaxation of Architectures
    - (4-1-1) Cell Architecture
    - (4-1-2) Network Architecture
  - (4-2) Optimization
  - (4-3) Decoding Discrete Architectures
- (5) Experimental Results
  - (5-1) Architecture Search Implementation Details
  - (5-2) Semantic Segmentation Results
    - (5-2-1) Cityscapes
    - (5-2-2) PASCAL VOC 2012
    - (5-2-3) ADE20K
- (6) Conclusion

BibTeX
------

```bibtex
@inproceedings{liu2019auto,
  title={Auto-deeplab: Hierarchical neural architecture search for semantic image segmentation},
  author={Liu, Chenxi and Chen, Liang-Chieh and Schroff, Florian and Adam, Hartwig and Hua, Wei and Yuille, Alan L and Fei-Fei, Li},
  booktitle={Proceedings of the IEEE Conference on Computer Vision and Pattern Recognition},
  pages={82--92},
  year={2019}
}
```

See Also
----------

- GitHub: https://github.com/tensorflow/models/tree/master/research/deeplab
